Makefile

```diff
@@ -196,6 +196,17 @@ prepublish:
 	IS_PUBLISH=true $(MAKE) test
 
 new-version:
+	@echo "!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!"
+	@echo "!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!"
+	@echo "!!!!!!                                                   !!!!!!"
+	@echo "!!!!!!   Update classStaticPrivateFieldDestructureSet    !!!!!!"
+	@echo "!!!!!!   helper version in                               !!!!!!"
+	@echo "!!!!!!   packages/babel-helpers/src/helpers.js           !!!!!!"
+	@echo "!!!!!!   packages/babel-helper-create-class-features-plugin/src/fields.js"
+	@echo "!!!!!!                                                   !!!!!!"
+	@echo "!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!"
+	@echo "!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!"
+	@exit 1
```

Comment:
We also need to disable the release action manual trigger since it doesn't use this `new-version` command, just to be sure :grimacing: